examples/templates/example.html

```diff
@@ -198,6 +197,9 @@ <h5 class="source-heading">index.html</h5>
     &lt;script src="https://cdn.polyfill.io/v3/polyfill.min.js?features=fetch,requestAnimationFrame,Element.prototype.classList,TextDecoder"&gt;&lt;/script&gt;
     &lt;script src="https://cdnjs.cloudflare.com/ajax/libs/core-js/3.18.3/minified.js"&gt;&lt;/script&gt;{{#if extraHead.remote}}
 {{ indent extraHead.remote spaces=4 }}{{/if}}
+    &lt;style&gt;
+    @import "node_modules/ol/ol.css";
+    &lt;/style&gt;
```

Comment:
To make this work in codesandbox.io ("Edit" button), you'll have to change it to
```html
&lt;link rel="stylesheet" href="node_modules/ol/ol.css"&gt;
```

Reply:
Thanks @ahocevar! I just made suggested change.